# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipsec-tools/checkpointVPNquirk.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ipsec-tools-0.6.5/src/racoon/oakley.c.orig	2006-02-11 14:13:53.183921968 +0100
+++ ipsec-tools-0.6.5/src/racoon/oakley.c	2006-02-11 14:15:35.584354736 +0100
@@ -2189,8 +2189,8 @@
 	case ISAKMP_CERT_CRL:
 	default:
 		plog(LLV_ERROR, LOCATION, NULL,
-			"Invalid CR type %d\n", type);
-		return -1;
+			"Warning: Ignoring invalid CR type %d\n", type);
+		return 0;
 	}
 
 	*c = save_certbuf(gen);
